CHANGELOG.md

```diff
@@ -11,8 +11,34 @@
   - `jest/prefer-to-be-null`
   - `jest/prefer-to-be-undefined`
   - `jest/valid-expect`
-* Added `shopify/webpack` plugin
-* Added `shopify/webpack/no-unnamed-dynamic-imports` rule
+* `yarn prettier` now prettifies source files
+* `shopify/webpack` plugin
+* `shopify/webpack/no-unnamed-dynamic-imports` rule
+
+### Changed
+* Updated dependencies to their latest versions (full details in [#63](https://github.com/Shopify/eslint-plugin-shopify/pull/63))
+* Updated prettier to 1.9.2, introducing a change in function parens style (set to `arrowParens: 'always'`):
+
+    ```js
+    // Before
+    const foo = myArray.map(foo => {});
+
+    // After
+    const foo = myArray.map((foo) => {});
+    ```
+
+    #### ⚠️ Upgrade path
+
+    Your project config files (`package.json`, `.prettierrc`, `.eslintrc`…)
+    may need to be updated like so:
+
+    ```diff
+       "singleQuote": true,
+       "bracketSpacing": false,
+       "trailingComma": "es5",
```

Comment:
Our config uses `trailingComma: 'all'`

Reply:
Ha, is that something we should also standardize? Looks like not all codebases use it like that.

Reply:
Pretty sure it is consistent, our listing rules have always enforced trailing commas.

Reply:
Which code bases don't use it? They should be unless they have a good reason not too. 

In the case of `eslint-plugin-shopify`, the repo itself uses es5 so we don't break consumers who may be on older versions of node. (eg. u2). We could add a build step in this repo to solve that, but don't think it's worth the effort.